### ISSUE TITLE
Run classifier? on augmented dataset

### ISSUE THREAD

**Issue by viggo-gascou** [viggo-gascou/fyp2022p03GroupB2#8]
Run at least one experiment with an augmented dataset that includes additional data selected based on training data perplexity under a language model trained the task you've selected. 
Possible sources of extra data include:

* the training sets of the other tasks,
* the Broad Twitter Dataset [here](https://github.com/GateNLP/broad_twitter_corpus)
* or any other dataset you can think of.

To obtain labels for the augmented data, you could try fully automatic annotation, 
or (full or partial) human validation of labels.
